src/sentry/flags/docs/api.md

```diff
@@ -83,7 +83,26 @@ Retrieve a single flag log instance.
   }
   ```
 
-## Signing Secret [/organizations/<organization_id_or_slug>/flags/hooks/provider/<provider>/signing-secret/]
+## Signing Secrets [/organizations/<organization_id_or_slug>/flags/hooks/signing-secrets/]
+
+### Browse Signing Secrets [GET]
+
+Browse a list of signing secrets. Secrets are unique per provider. Secrets only show the first six characters; the remainder are redacted.
+
+- Response 200
+
+  ```json
+  {
+    "data": [
+      {
+        "createdAt": "2024-12-12T00:00:00Z",
+        "createdBy": 12345,
+        "provider": "launchdarkly",
+        "secret": "abc123**********"
+      }
+    ]
+  }
+  ```
 
 ### Create Signing Secret [POST]
```

Comment:
is the post endpoint different now? i see `/<provider>/` was removed

Reply:
Yes! I neglected to add provider as a field in the payload.

Reply:
ok cool, i'll need to update this in the sidebar then👍